Automattic/pocket-casts-ios
# [Custom Playback Settings]  Feature Flag & Segmented Control

| 📘 Part of: #2253
|:---:|

Fixes #2254

This PR adds the `customPlaybackSettings` Feature Flag and the Segmented Control into the Effects Player VC

| FF Off | FF On |
| -------- | ------- |
| ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-11_14 54 02](https://github.com/user-attachments/assets/cc4cb9bb-96ae-496c-9fd6-2ea2a681bc09) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-14_12 06 17](https://github.com/user-attachments/assets/d42f9d8e-417e-4615-8cac-2dbbfe7b4740) |



## To test

> [!NOTE]
> FF is `customPlaybackSettings`

- CI must be 🟢 
- Enable the FF
- Play an episode and open the Effects VC
- Check the Segmented appears at the top
- Disable the FF
- Re-open the Effects VC and check everything works as expected
- Confirm in both scenarios there are no autolayout warnings

## Checklist
